user-guide/Advanced_Functionality/Databases/Indexing_database/Migrating_from_Elasticsearch_to_Opensearch.md

```diff
@@ -0,0 +1,205 @@
+
+Using Kibana, you can restore the snapshot in the following way:
+
+1. Check the *path.repo* configuration in *elasticsearch.yml*.
+
+1. Check the existing repositories by sending the following request:
+
+   ```txt
+   GET /_snapshot/_all
+   ```
+
+1. Create the repository by sending the following request:
+
+   ```txt
+    PUT /_snapshot/<repo_name>
+    {
+        "type": "fs",
+        "settings": {
+        "location" : "<shared_repo_path>",
+            "maxRestoreBytesPerSec" : "40mb",
+            "readonly" : "false",
+            "compress" : "true",
+            "maxSnapshotBytesPerSec" : "40mb"
+        }
+    }
+   ```
+
+1. Restore the snapshot by sending the following request:
+
+   ```txt
+    POST /_snapshot/<repo_name>/<snapshot_name>/_restore 
+   ```
+
+1. Check the snapshot by sending the following request:
+
+   ```txt
+    GET /_snapshot/<repo_name>/<snapshot_name>/_status
+   ```
+
+1. Check the cluster health by sending the following request:
+
+   ```txt
+    GET /_cluster/health
+   ```
+
+   The status of the cluster should turn green after the restore.
+
+## Run the re-indexing tool and take a snapshot
+
+1. Open a terminal, and go to the folder containing the tool. By default, this is the folder `C:\Skyline DataMiner\Tools\ReIndexElasticSearchIndexes`:
+
+   ```txt
+    cd C:\Skyline DataMiner\Tools\ReIndexElasticSearchIndexes
+   ```
+
+1. Run the tool with the following arguments:
+
+   | Argument | Description |
+   |----------|-------------|
+   | -Node or -N | The name of the node to be used for re-indexing (mandatory).<br>Format: `http(s)://127.0.0.1:9200` or `http(s)://fqdn:9200` |
+   | -User or -U | The user name, to be provided in case Elasticsearch was hardened.<br>See [Securing the Elasticsearch database](xref:Security_Elasticsearch) |
+   | -Password or -P | The user password |
+   | -DBPrefix or -D | The database prefix, to be provided in case a custom database prefix is used instead of the default `dms-` prefix.<br>If you do not provide a prefix, the default `dms-` will be used. |
+   | -TLSEnabled or -T | Whether or not TLS is enabled for this ElasticSearch database.<br>Values: true or false. Default: false |
+
+1. Take a snapshot of the re-indexed data by sending the following request:
+
+   ```txt
+   PUT /_snapshot/<repo_name>/<snapshot_name_reindexed>
+    {
+      "indices": "dms*"
+    }
+   ```
+
+1. Check the snapshot by sending the following request:
+
+   ```txt
+    GET /_snapshot/<repo_name>/<snapshot_name>/_status
+   ```
+
+   This request will return information about the status of the specified snapshot. The status should be "SUCCESS".
+
+## Restore the snapshot with the re-indexed data to a OpenSearch 2.11.1 cluster
+
+1. Check the *path.repo* configuration in *opensearch.yml*.
+
+   This configuration should point to a shared file system location to which each node has access.
+
+1. Check the existing repositories by sending the following request:
```

Comment:
What is the purpose of this check? What should the result be?

Reply:
added info. This request will return all registered snapshots in the cluster. In case the desired repository already exists the next step can be skipped.